bundle/edu.gemini.itc.shared/src/main/java/edu/gemini/itc/shared/ConfigCreator.java

```diff
@@ -154,10 +154,9 @@ public final ConfigCreatorResult createGhostConfig(final GhostParameters ghostPa
         final ConfigCreatorResult result = createCommonConfig(numExp);
         for (final Config step : result.getConfig()) {
             step.putItem(InstInstrumentKey, SPComponentType.INSTRUMENT_GHOST);
-            step.putItem(CcdXBinning, ghostParameters.spectralBinning());
-            step.putItem(CcdYBinning, ghostParameters.spatialBinning());
+            step.putItem(CcdXBinning, ghostParameters.binning());
             step.putItem(ReadModeKey, ghostParameters.readMode());
-            step.putItem(DetectorManufacturerKey, (GhostType.DetectorManufacturer.DEFAULT));
+            step.putItem(DetectorManufacturerKey, (DetectorManufacturer.DEFAULT));
```

Comment:
I may be confused about this, but I'm guessing `DetectorManufacturer` comes from copying from GMOS.  In GMOS we swapped the detectors from E2V to Hamamatsu and introduced a `DetectorManufacturer` to keep up with the difference.  There is no such difference in GHOST (I assume) so I'm confused about why there is a GHOST detector manufacturer enum.   There are blue and red detectors, but the manufacturer is presumably the same.  Maybe rename to `Detector` or `GhostDetector` if we need this?

Reply:
Good question, there are different data files for these "Detectors" maybe  we are talking about red and blue?


bundle/edu.gemini.itc/src/main/resources/ghost/ghost_E2V_CCD231-84-1-G57.dat
bundle/edu.gemini.itc/src/main/resources/ghost/ghost_E2V_CCD231-C6-1-G58.dat

Reply:
Actually those files seem identical. We could use just one and remove the whole `Detector` class

Reply:
Both files are the same because we only had the QE of one as good data. It was left separately in case it became available. 
As you can see in Ghost both detectors are from the same manufacturer but not the model, that was what I saw in the documentation.